cmd/xl-v1-healing.go

```diff
@@ -31,6 +31,15 @@ func (xl xlObjects) ReloadFormat(ctx context.Context, dryRun bool) error {
 	return NotImplemented{}
 }
 
+func (xl xlObjects) FixDeploymentID(ctx context.Context) error {
+	logger.LogIf(ctx, NotImplemented{})
+	return NotImplemented{}
+}
+
+func (xl xlObjects) GetDeploymentID() (string, error) {
+	return "", NotImplemented{}
+}
+
```

Comment:
xlObjects is does not implement ObjectInterface

Reply:
Removing both these methods.